packages/@lwc/engine/src/framework/__tests__/watcher.spec.ts

```diff
@@ -57,8 +57,9 @@ describe('watcher', () => {
             document.body.appendChild(elm);
             elm.updateRound();
 
-            Promise.resolve().then(_ => {
-                expect(counter).toBe(2);
+            return Promise.resolve().then(_ => {
+                // parent slotted into child, not need to rerender child
+                expect(counter).toBe(1);
```

Comment:
this test was passing before the refactor on synthetic shadow split, in which case, slotted elements were in fact triggrering redirect on the child element, but after we move to use native shadow on native, it does not longer re-render the child, and the test must be change accordingly.  At some point, this should be move to karma. I don't want to put too much effort on that just yet because I'm planning to change the slotting mechanism to match native shadow, to avoid rerendering child element when slotting.